src/pb_controller.cpp

```diff
@@ -371,7 +371,9 @@ void pb_controller::play_file(const std::string& file) {
 	cmdline.append(utils::replace_all(file,"\"", "\\\""));
 	cmdline.append("\"");
 	stfl::reset();
-	utils::run_interactively(cmdline, "pb_controller::play_file");
+	// Is the following exit code important?
```

Comment:
I think it *is* important. I just tried with MPV, and it returns 2 if file doesn't exist, but 0 if I exit normally. Looks like showing a "Player failed to open the file" error would be okay here.